CMakeLists.txt

```diff
@@ -1088,6 +1088,7 @@ target_link_libraries(address_sorting
 if(gRPC_INSTALL)
   install(TARGETS address_sorting EXPORT gRPCTargets
     RUNTIME DESTINATION ${gRPC_INSTALL_BINDIR}
+    BUNDLE DESTINATION  ${gRPC_INSTALL_BINDIR}
```

Comment:
the CMakeLists.txt is autogenerated from templates/CMakeLists.txt.template, so only updating this file won't work.

Reply:
Oh, I see :)

Reply:
Should I update both or is the generated CMakeLists.txt updated by the CI/CD?